### PR TITLE
Replace environment variable with specific path

### LINKED ISSUE #33: Service doesn't update status when using Env:AppData
The change made in [this commit](https://github.com/EBOOZ/TeamsStatus/commit/d100c67b346dd78c0cecb3a48ccabb2325e8be62) break the service unless it is running under your actual user account. By default the service runs as `LocalSystem` so cannot find the teams log file.

Reverting this change fixed the service for me.

### PR DESCRIPTION
**Pull request by bmdixon**
The service runs as LocalService so the $env:APPDATA path is incorrect.

Resolves #33

### CODE REVIEW

File: Get-TeamsStatus.ps1 [hunk 1]
```diff
@@ -66,7 +66,7 @@ $TeamsProcess = Get-Process -Name Teams -ErrorAction SilentlyContinue
 
 # Check if Teams is running and start monitoring the log if it is
 If ($null -ne $TeamsProcess) {
-    If($TeamsStatus -eq $null){ }
+    If ($null -eq $TeamsStatus) { }
```
Comment (bmdixon) [inline Get-TeamsStatus.ps1:69]: It's best practice in PowerShell to have the `$null` on the left of checks - See [here](https://docs.microsoft.com/en-us/powershell/scripting/learn/deep-dives/everything-about-null?view=powershell-7.1#checking-for-null)

**Comment by metbril** [comment]
This PR basically reverts #28?

**Comment by bmdixon** [comment]
> This PR basically reverts #28?

yes, the change made in #28 means the log file directory is wrong.

**Comment by felipecrs** [comment]
This needs to be merged, otherwise the service does not work. I didn't find any other way around, as nssm does not allow to set the user to start the service without asking for its password.

**Comment by metbril** [comment]
Instead of hardcoding the path, would it be an option to test if the `$env:APPDATA` variable returns anything and if not use the hardcoded path instead? That would with both use cases (running as LocalSystem or user)

**Comment by felipecrs** [comment]
It should be. I suppose the most difficult part is to actually test what's inside $env:APPDATA when running as a service.

**Comment by metbril** [comment]
Google lead me here:

https://social.msdn.microsoft.com/Forums/windows/en-US/1889f615-0a88-4c6e-9c19-f93b00846a9d/get-special-folder-on-windows-service-running-as-local-system

One could test against this result. But i don't mind a hardcoded path.

**Comment by chriscolden** [comment]
This is what i've come up with.

```
If ($env:APPDATA -match "C:\\Users") {
	$path = "$env:APPDATA\Microsoft\Teams\logs.txt"
}
Else {
	$path = "C:\Users\$UserName\AppData\Roaming\Microsoft\Teams\logs.txt"
}
```

Then use 

`Get-Content -Path $path`

**Comment by EBOOZ** [comment]
I've updated the script to work with the current Teams client. Closing this one to clean up.

**Pull request closed without merging**